lib/plugins/deploy-interactive.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const ora = require('ora');
+
+class InteractiveDeployProgress {
+  constructor(serverless) {
+    this.serverless = serverless;
+    this.deployProgress = ora('Deploying your project...');
+    this.packageProgress = ora('Packaging your project...');
```

Comment:
How does it look visually, do we like `ora` styles?

Reply:
Switched to `cli-progress-footer`